app/Http/Controllers/CoverageController.php

```diff
@@ -158,12 +158,12 @@ public function manageCoverage(): View|RedirectResponse
 
         // Assign last author
         if (isset($_POST['assignLastAuthor'])) {
-            $CoverageFile2User->AssignLastAuthor(intval($buildid));
+            $CoverageFile2User->AssignAuthors($buildid, true);
```

Comment:
Since we're now using php 8.x, it might be nice to name the second parameter here for clarity, so that someone reading this line won't have to go and look for what it stands for. E.g., something like:
```suggestion
            $CoverageFile2User->AssignAuthors($buildid, onlylast: true);
```

Reply:
I think it seems reasonable to do here, so I have made the change.  We should do some thinking about whether we want this to set a precedent though.

Named arguments are primarily to let you specify an optional argument without specifying the rest.  Something like this for example:
```php
function foo($a = 1, $b = 2, $c = 3) {}
```
You can use named arguments here to only specify `$c`, when you'd otherwise have to specify the default values for the first two parameters.

The downside of using named arguments is that if you want to change the name of a given parameter--something which should be a purely internal change--you then have to go to all of the call sites and change the name of the parameter there as well.  Any good IDE should be able to automatically identify parameters.  For example, here's what PHPStorm looked like before I made the change:
![image](https://github.com/Kitware/CDash/assets/16820599/cb579b42-118e-442f-b9f0-09acf7c73d05)